proton_driver/util/compat.py

```diff
@@ -5,6 +5,18 @@
 except ImportError:
     import dummy_threading as threading  # noqa: F401
 
+import json  # noqa: F401
+try:
+    import orjson as json  # noqa: F811
```

Comment:
why import orjson / ujson? do we have a preference so could try-import them in order; only import if more preferred lib not exsts.